### DISCUSSION TITLE
Append popover to the body

### DISCUSSION THREAD

**Discussion by juanpablob** [floating-ui/floating-ui#1278] [Q&A]
Hello there! Has popperjs any option to make the popper be appended to the body? I have div with overflow hidden (horizontal scroll filters for mobile) with buttons that open a dropdown (popper) and of course the popper remains hidden because its parent has overflow hidden, therefore append the popper to the body is a good idea, for this scenario and for other I've encountered.

Does anyone have any idea? I appreciate it! Thanks in advance!

**Comment by FezVrasta** [comment]
Hi, Popper doesn't mutate the DOM, but you can place your tooltip element anywhere you want and the library will take care to position it
